### ISSUE TITLE
How to (de-)activate specific warnings on the command line?

### ISSUE THREAD

**Issue by GitMensch** [LADSoft/OrangeC#216]
I did not found anything about warnings options in https://orangec.readthedocs.io/en/latest/occ/OCC%20Command%20Line/

In general it would be nice to have the options (see also [GCC warning options](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html) / [MS C warning options](https://msdn.microsoft.com/en-us/library/thxezb7y.aspx)):

* disable all warnings (`-w`)
* disable a specific warning `-wd*number*`
* specify that a specific warning is to be counted as error (`-we*number*`)
* specify that all warnings are to be counted as error (`-Werror` / `-wx`)
* optional: report a specific warning only once `-wo*number*`
* enable a specific warning `-w*number*`

**Comment by GitMensch** [comment]
Thank you for implementing the new options (nearly as noted above) and providing [https://github.com/LADSoft/OrangeC/blob/master/doc/occ/OCC%20Warning%20Control.md].

Just two questions instead if a request to reopen:
* Wouldn't it be good to add `/w+` for the additional warnings instead of `/E+number` where, according to the docs a number needs to be specified?
* You very likely have tested it, can I suggest to add testing code to the codebase and do the test from there? This way you ensure that all is fine, also in the future (tests likely would be:  `/weNONUM`, `/we9999` [not existing], `/we9999999999999999999999999999999` [no int], `/w /wo23` [only warning 123 active] + tests about actual warnings).

**Comment by LADSoft** [comment]
I was trying to leave /E alone, but adding /w+ sounds good!   I will add test code too.
